docs/_static/flowchart_ewatercycle.excalidraw.svg

```diff
@@ -12,6 +12,6 @@
 stroke-width="1" fill="none"></path></g></a><g transform="translate(778.4333318074544 1540.3333333333333) rotate(0 125.9000015258789 25)"><text x="125.9000015258789" y="0" font-family="Virgil, Segoe UI Emoji" font-size="20px" fill="#1e1e1e" text-anchor="middle" style="white-space: pre;" direction="ltr" dominant-baseline="text-before-edge">Create a custom forcing </text><text x="125.9000015258789" y="25" font-family="Virgil, Segoe UI Emoji" font-size="20px" fill="#1e1e1e" text-anchor="middle" style="white-space: pre;" direction="ltr" dominant-baseline="text-before-edge">generator</text></g><g stroke-linecap="round"><g transform="translate(773.6369834720574 1649.7777777777778) rotate(0 -21.065301313188115 29.961560851222885)"><path d="M0.43 0.02 C-6.7 10.14, -34.87 50.79, -42.17 60.94 M-0.8 -1.01 C-8.06 8.7, -35.32 48.37, -42.56 58.94" stroke="#1e1e1e" stroke-width="1" fill="none"></path></g><g transform="translate(773.6369834720574 1649.7777777777778) rotate(0 -21.065301313188115 29.961560851222885)"><path d="M-35.01 31.67 C-37.26 39.88, -39.91 51.36, -42.51 58.29 M-34.59 30.28 C-36.7 37.19, -39.79 43.63, -41.97 59.84" stroke="#1e1e1e" stroke-width="1" fill="none"></path></g><g transform="translate(773.6369834720574 1649.7777777777778) rotate(0 -21.065301313188115 29.961560851222885)"><path d="M-18.1 43.3 C-26.34 47.46, -34.96 54.84, -42.51 58.29 M-17.68 41.91 C-23.95 45.99, -31.29 49.51, -41.97 59.84" stroke="#1e1e1e" stroke-width="1" fill="none"></path></g></g><mask></mask><g mask="url(#mask-q6Yk4z02mV9-hb7e8QEXw)" stroke-linecap="round"><g transform="translate(602.9596408341088 1455.062541509576) rotate(0 71.77962818660325 26.71820664742188)"><path d="M-0.02 -0.69 C24.08 8.08, 121.18 44, 145.05 52.87 M-1.49 1.57 C22.49 10.47, 120.07 45.48, 144.58 54.12" stroke="#1e1e1e" stroke-width="1" fill="none"></path></g><g transform="translate(602.9596408341088 1455.062541509576) rotate(0 71.77962818660325 26.71820664742188)"><path d="M113.43 55.82 C123.8 52.64, 133.43 53.73, 145.95 54.5 M114.88 54.85 C123.4 54.25, 133.63 53.85, 144.68 53.9" stroke="#1e1e1e" stroke-width="1" fill="none"></path></g><g transform="translate(602.9596408341088 1455.062541509576) rotate(0 71.77962818660325 26.71820664742188)"><path d="M120.32 36.49 C128.4 39.84, 135.71 47.45, 145.95 54.5 M121.77 35.52 C128 41.09, 136.01 46.93, 144.68 53.9" stroke="#1e1e1e" stroke-width="1" fill="none"></path></g></g><mask id="mask-q6Yk4z02mV9-hb7e8QEXw"><rect x="0" y="0" fill="#fff" width="847.1111111111111" height="1608.4785642868758"></rect><rect x="654.8270420034937" y="1469.270552898226" fill="#000" width="40.41666793823242" height="25" opacity="1"></rect></mask><g transform="translate(654.8270420034937 1469.270552898226) rotate(0 19.9122270172183 12.510195258771887)"><text x="20.20833396911621" y="0" font-family="Virgil, Segoe UI Emoji" font-size="20px" fill="#1e1e1e" text-anchor="middle" style="white-space: pre;" direction="ltr" dominant-baseline="text-before-edge">No...</text></g></svg>
```

Comment:
There's a link to https://ns.nl in here now!

Reply:
doh, link removed. 

Next up fix the rtd build, seems rtd is no longer installing bunch of packages, see https://readthedocs.org/projects/ewatercycle/builds/22129307/